activities/organization/finance-updates/finance-update-september-2020.md

```diff
@@ -0,0 +1,32 @@
+---
+type: resource
+---
+
+# Finance update September 2020
+
+## Summary
+
+* Current cash at bank position: € 1,2 million
+* Cost incurred (cash burned):  € 344k (2019) + € 542k (till Aug 2020) = € 886.569
+* Projected runway:  1.200.000 / 88.000 = 13 months (current cash position / forecasted cost)
+
+## Funding
+
+The Foundation started with a startup-loan of € 1 million (€ 0,2mio + 0,8mio). This loan (including accrued interest € 29.691) is paid back on the 24th August 2020. On the 24th August the Foundation received a loan of € 1.702.200 that is part of a fiscal gift schema. Furthermore, the first (formal fiscal) gift was received € 435.500. In total an net-amount of € 1.137.700 (€ 1,7mio + 0,4mio -/- 1mio) was received.
+
+## State of the administration
+
+* Budget sheet: the budget sheet is updated with the latest insights (new fiscal gift-loan, elimination of the amortization schemes, update of personnel cost)
+* General ledger (Exact): Updated
+* VisionPlanner: updated (interfaced with Exact)
+
+## Interim conclusion
+
+The new funding loan came in time for the funding of the foundation. The so called runway is extended; the Opex (operational expenses) can be funded for the upcoming 12 to 13 months. Even though the runway is extended by AS, there is a need to get inflow of cash by:
```

Comment:
Also: see comment on seemingly identical paragraph in Dec 2020 re AS not being self explanatory.
```suggestion
The new funding loan came in time for the funding of the Foundation for Public Code. The runway is extended; the Opex (operational expenses) can be funded for the upcoming 12 to 13 months. Even though the runway is extended by AS, there is a need to get inflow of cash by:
```